install/installer/pkg/common/common.go

```diff
@@ -339,7 +339,7 @@ func Affinity(orLabels ...string) *corev1.Affinity {
 	}
 }
 
-func RepoName(repo, name string) string {
+func RepoName(repo, name string, cfg *config.Config) string {
```

Comment:
It's not actually. The `repo` is wherever the original image is hosted, which might be Gitpod repo or `docker.io` (or something else). The other one is what the value in the `config.yaml` is.

As when this was raised by @nandajavarma, I used the Config struct to avoid having to change this in future.

Reply:
We introduced the `RepoName` for exactly this reason: swap out the name of a repo with something set in the config. Looking at how `RepoName` is called, it's usually called with `RepoName(cfg.Repository, ...)` (through `ImageName`). There are cases where third-party repos are involved, but that's encapsulated in the `ThirdParty` repo function already.